### DISCUSSION TITLE
link to folder path

### DISCUSSION THREAD

**Discussion by pyinstallerbaggins** [pallets/click#2780] [Q&A] [answered]
Is there a way to take a folder path and create a hyperlink to that folder path in the CLI

Something like
console.print(f'Your file(s) is saved here -> [link=C:\Account\Consultant Reporting\]Here[/link]!

**Comment by davidism** [comment, marked as answer]
That's up to whether your terminal manipulates output, not something Click can control.

**Reply by pyinstallerbaggins** [reply]
i just used click.launch. sufficed

thank you!
